TODO

```diff
@@ -1,23 +0,0 @@
-consider stripping vdoms for VERPs; tnx PJH
-consider ~ in qmail-local for doing defaultdelivery (not recursively)
-consider POP bulletins
-turn qmail-upq into a more serious queue-moving utility
-consider fast-greeting option in qmail-smtpd
-build a returnmail package
-
-expand strerr coverage
-redo control interface
-allow concurrency over 255
-allow more channels at compile time
-test for linux fifo close bug at compile time
-
-eliminate qsmhook
-finish OTBS conversion
-use mess822 in qmail-inject
-use mess822 in qreceipt
-use mess822 in qbiff
-use mess822 in maildirwatch
-eliminate token822, headerbody, hfield
```

Comment:
At a glance, `token822.h` looks like the sort of API that `mess822.h` replaces. So this is probably redundant. Since we have #88, I'm comfortable not worrying further about this.